vuepress/docs/docs/federation.md

```diff
@@ -0,0 +1,81 @@
+# Federation
+
+**Federation** is an optional module which can be included in your configuration to enroll with a federated schema.
+
+## Dependencies
+
+`caliban-federation` only depends on `caliban-core` and is very unobtrusive.
+
+To use, add the following line to your `build.sbt` file:
+
+```
+libraryDependencies += "com.github.ghostdogpr" %% "caliban-federation" % "0.7.3"
+```
+
+## Federating
+
+Federation allows graphs to become part of a larger graph without having to share models or create brittle
+schema stitching code at the gateway level.
+
+You can read more about federation and why it may be useful [here](https://www.apollographql.com/docs/apollo-server/federation/introduction/)
```

Comment:
```suggestion
You can read more about federation and why it may be useful [here](https://www.apollographql.com/docs/apollo-server/federation/introduction/).
```